<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1612899571933" stop="1612899596781" version="1.5.4" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Default suite : Default test</name>
    <title>Default suite : Default test</title>
    <test-cases>
        <test-case start="1612899588050" stop="1612899588166" status="broken">
            <name>TC_08_Navigate_to_AdminLogin</name>
            <description type="text">Navigate to Admin Login page</description>
            <failure>
                <message>FileNotFoundException: C:\Users\sukki\OneDrive\EzSchedulerscratch\EzScheduler\inputFiles\Admin.xlsx (The process cannot access the file because it is being used by another process)</message>
                <stack-trace>java.io.FileNotFoundException: C:\Users\sukki\OneDrive\EzSchedulerscratch\EzScheduler\inputFiles\Admin.xlsx (The process cannot access the file because it is being used by another process)&#13;
	at java.io.FileInputStream.open0(Native Method)&#13;
	at java.io.FileInputStream.open(FileInputStream.java:195)&#13;
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:138)&#13;
	at java.io.FileInputStream.&lt;init&gt;(FileInputStream.java:93)&#13;
	at com.sailotech.ezscheduler.Implpages.Adminlogin.AdminLogin(Adminlogin.java:28)&#13;
	at com.sailotech.ezscheduler.Testpages.AdminUnverifiedDoctorsTest.TC_08_Navigate_to_AdminLogin(AdminUnverifiedDoctorsTest.java:49)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)&#13;
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)&#13;
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.sailotech.ezscheduler.Testpages.AdminUnverifiedDoctorsTest"/>
                <label name="testMethod" value="TC_08_Navigate_to_AdminLogin"/>
                <label name="severity" value="normal"/>
                <label name="testId" value="TC_08"/>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="12468@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1612899588801" stop="1612899588804" status="canceled">
            <name>TC_09_AdminVerifyDoctor</name>
            <description type="text">After Registration Admin is Verifying the Unverified Doctors</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:186)&#13;
	at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:61)&#13;
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1389)&#13;
	at org.testng.internal.Invoker.registerSkippedTestResult(Invoker.java:1063)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.sailotech.ezscheduler.Testpages.AdminUnverifiedDoctorsTest"/>
                <label name="testMethod" value="TC_09_AdminVerifyDoctor"/>
                <label name="severity" value="normal"/>
                <label name="testId" value="TC_09"/>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="12468@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1612899588813" stop="1612899588814" status="canceled">
            <name>TC_10_AdminLogout</name>
            <description type="text">Admin Logout from EzScheduler Website</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:425)&#13;
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:186)&#13;
	at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:61)&#13;
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1389)&#13;
	at org.testng.internal.Invoker.registerSkippedTestResult(Invoker.java:1063)&#13;
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)&#13;
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)&#13;
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)&#13;
	at org.testng.TestRunner.privateRun(TestRunner.java:648)&#13;
	at org.testng.TestRunner.run(TestRunner.java:505)&#13;
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)&#13;
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)&#13;
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)&#13;
	at org.testng.SuiteRunner.run(SuiteRunner.java:364)&#13;
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)&#13;
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)&#13;
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)&#13;
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)&#13;
	at org.testng.TestNG.runSuites(TestNG.java:1049)&#13;
	at org.testng.TestNG.run(TestNG.java:1017)&#13;
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)&#13;
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)&#13;
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)&#13;
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="Default suite"/>
                <label name="testGroup" value="Default test"/>
                <label name="testClass" value="com.sailotech.ezscheduler.Testpages.AdminUnverifiedDoctorsTest"/>
                <label name="testMethod" value="TC_10_AdminLogout"/>
                <label name="severity" value="normal"/>
                <label name="testId" value="TC_10"/>
                <label name="host" value="DESKTOP-A4GJKOM"/>
                <label name="thread" value="12468@DESKTOP-A4GJKOM.main(1)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
